package-lock.json

```diff
@@ -3868,6 +3868,7 @@
         "spawn-promise": "^0.1.8",
         "style-loader": "^0.23.1",
         "svg-url-loader": "^2.3.2",
+        "terser-webpack-plugin": "^1.2.1",
```

Comment:
All those changes in `package-lock.json` seem unrelated. Could you clarify why they're here, or remove them otherwise? :)

Reply:
Hmm, nothing intentional, just the result of running `npm install`. I'll try to merge in `develop` and see if it persists…